### PR TITLE
Add a related_file to the issue

### PR DESCRIPTION
**Pull request by antoine-de**
linked to https://github.com/rust-transit/gtfs-structure/pull/52

Add a related_file section in the response when we know the position in a file of the error.

it adds a `related_file` section:
```json
"related_file": {
    "file_name": "stop_times.txt",
    "line": 11,
"headers": [
              "stop_id",
              "stop_name",
              "stop_desc",
              "stop_lat",
              "stop_lon",
              "zone_id",
              "stop_url",
              "location_type",
              "parent_station"
            ],
            "values": [
              "stop_with_bad_coord",
              "Moo",
              "",
              "baaaaaad_coord",
              "-116.40094",
              "",
              "",
              "",
              "1"
            ]
},
```

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -24,3 +24,4 @@ structopt-derive = "0.3"
 url = "2"
 actix-web = "2"
 actix-rt = "1"
+csv = "1"
```
Comment (Tristramg) [inline Cargo.toml]: do we really need a dependency on csv? Can’t gtfs-structure use its own error, or re-export it?

File: src/validate.rs [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
-use itertools::Itertools;
 use serde::Serialize;
 use std::collections::BTreeMap;
+use std::error::Error;
 
-fn pretty_print_error(error: anyhow::Error) -> String {
-    error.chain().map(|c| format!("{}", c)).join(": ")
+fn create_unloadable_model_error(error: &gtfs_structures::Error) -> issues::Issue {
+    let mut msg = format!("{}", error);
+    if let Some(inner) = error.source() {
```
Comment (Tristramg) [inline src/validate.rs]: Not sure, just suggesting, maybe something like:

```
let msg = if let Some(…) { format("{} : {}", error, inner) } else { format!("{}", error;}
```

**Review by Tristramg — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (antoine-de) [reply Cargo.toml]: oups you're right I don't need it anymore

File: src/validate.rs (hunk 2 above)
Comment (antoine-de) [reply src/validate.rs]: yep, sure it's better!

**Pull request merged**